packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -58,6 +71,22 @@ export class FauxClassGenerator {
     ['complexvalue', 'Object']
   ]);
 
+  constructor(emitter: EventEmitter, cancellationToken?: CancellationToken) {
+    this.emitter = emitter;
+    this.cancellationToken = cancellationToken;
+  }
+
+  private errorExit(errorMessage: string): Promise<string> {
+    this.emitter.emit(LocalCommandExecution.STDERR_EVENT, errorMessage);
+    this.emitter.emit(LocalCommandExecution.ERROR_EVENT, '1');
+    return Promise.reject(`${FAILURE_CODE} - ${errorMessage}`);
+  }
+
+  private successExit(): Promise<string> {
+    this.emitter.emit(LocalCommandExecution.EXIT_EVENT, '0');
```

Comment:
The exit even is also supposed to emit something similar to https://nodejs.org/api/child_process.html#child_process_event_exit

Could you check that you are emitting the right signatures if you want to use this technique?